experiments/functional/zfs-LocalPV/zfspv-snapshot/test.yml

```diff
@@ -0,0 +1,158 @@
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+    - /mnt/parameters.yml
+    
+  tasks:
+    - block:
+
+        ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: "/utils/fcm/update_litmus_result_resource.yml"
+          vars:
+            status: 'SOT'
+        
+        - block:
+            - name: Get the application pod name
+              shell: >
+                kubectl get pod -n {{ app_ns }} -l {{ app_label }} 
+                --no-headers -o custom-columns=:.metadata.name
+              args:
+                executable: /bin/bash
+              register: app_pod_name
+
+            - name: Check if the application pod is in running state
+              shell: >
+                kubectl get pods -n {{ app_ns }} -l {{ app_label }} 
+                --no-headers -o custom-columns=:.status.phase
+              args:
+                executable: /bin/bash
+              register: app_pod_status
+              failed_when: "'Running' not in app_pod_status.stdout"
+
+            - block:
+              - name: Create some test data into the application
+                include_tasks: "/utils/scm/applications/busybox/busybox_data_persistence.yml"
+                vars:
+                  status: 'LOAD'
+                  ns: "{{ app_ns }}"
+                  pod_name: "{{ app_pod_name.stdout }}"  
+              when: data_persistence == 'busybox'
+            
+            - block:
+              - name: Create some test data into the application
+                include_tasks: "/utils/scm/applications/mysql/mysql_data_persistence.yml"
+                vars:
+                  status: 'LOAD'
+                  ns: "{{ app_ns }}"
+                  pod_name: "{{ app_pod_name.stdout }}"  
+              when: data_persistence == 'mysql'
+                       
+            - name: Update the snapshot template with the test specific variables
+              template: 
+                src: snapshot.j2
+                dest: snapshot.yml
+
+            - name: create snapshot   
```

Comment:
are we checking if volumesnapshotclass exist

Reply:
Good point.
We've to check the volumesnapshotclass exist or not. will add this check condition

Reply:
Good point.
We've to check the volumesnapshotclass exist or not. will add this check condition

Reply:
Added !!